metatron-app/metatron-discovery
# #2961 improve query detail and datasource list in engine monitoring

### Description

Improve query detail and datasource list in engine monitoring

**Related Issue** : #2961 



### How Has This Been Tested?

1. Go to Engine Monitoring
2. Go to Query tab and check Query detail UI change.
3. Go to datasource tab and check the enable / disable / permanently delete function.
  a Change the data source that is in the enable status to disable.
  b Verify that it is changed to disable in the datasource list. (It may take a few minutes.)
  c Enable the disabled data source and confirm the status change.
  d Permanently delete the data source and make sure it is not visible in the data source list.
  e Using Retention, drop a specific interval or period.
  f Enable to check if the entire data is available.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
